Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationBadge.m

```diff
@@ -0,0 +1,40 @@
+/*
+* Backpack - Skyscanner's Design System
+*
+* Copyright 2018-2020 Skyscanner Ltd
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#import "BPKHorizontalNavigationBadge.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@implementation BPKHorizontalNavigationBadge
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    self.type = BPKBadgeTypeOutline;
+
+    return self;
+}
+
+- (void)setColor:(UIColor *)color {
+    if (_color != color) {
+        _color = color;
+
+        [super performSelector:@selector(changeContentColor:) withObject:color];
```

Comment:
Could we either:
1. Add a runtime check to make sure the value is available
Or
2. Implement methods in BPKBadge for getting the colour. Then we can just safely override them without changing the public API